templates/cli/lib/updater.js.twig

```diff
@@ -0,0 +1,98 @@
+const progress = require('cli-progress');
+const chalk = require('chalk');
+
+const SPINNER_ARC = 'arc';
+const SPINNER_DOTS = 'dots';
+
+const spinners = {
+    [SPINNER_ARC]: {
+        "interval": 100,
+        "frames": ["◜", "◠", "◝", "◞", "◡", "◟"]
+    },
+    [SPINNER_DOTS]: {
+        "interval": 80,
+        "frames": ["⠋", "⠙", "⠹", "⠸", "⠼", "⠴", "⠦", "⠧", "⠇", "⠏"]
+    }
+}
+
+class Updater {
```

Comment:
Since no variables are stored on class, can we make them into normal functions? Avoid the need to use class, and directly export the functions in module.exports